app/views/parsers/index.html.erb

```diff
@@ -24,8 +24,8 @@
           <td><%= parser.strategy %></td>
           <td><%= link_to parser.source.partner.name, edit_partner_path(parser.source.partner) %></td>
           <td><%= link_to parser.source.source_id, edit_source_path(parser.source) %></td>
-          <td><%= parser.versions.last.try(:created_at).try(:localtime)&.to_formatted_s(:long) %></td>
-          <td><%= parser.last_edited_by %></td>
+          <td><%= parser.try(:updated_at).try(:localtime)&.to_formatted_s(:long) %></td>
```

Comment:
Just double checking - but does the table head for this column make sense with the changes / not need to be renamed or anything?

Reply:
Na it's the same value :).